x/coinswap/types/msgs.go

```diff
@@ -72,3 +76,15 @@ func NewMsgRemoveLiquidity(
 		Sender:            sender,
 	}
 }
+
+func GetSignersFromMsgSwapOrderV2(msg protov2.Message) ([][]byte, error) {
+	msgv2, ok := msg.(*coinswapv1.MsgSwapOrder)
+	if !ok {
+		return nil, fmt.Errorf("invalid x/coinswap/MsgSwapOrder msg v2: %v", msg)
+	}
+
+	signers := [][]byte{}
+	signers = append(signers, []byte(msgv2.Input.Address))
```

Comment:
It seems safer to use an official account address converter like [StringToBytes](https://github.com/cosmos/cosmos-sdk/blob/19632293df2b59fc42ad7031d45116ada2a6b047/codec/address/bech32_codec.go#L26-L45) to convert to bytes and return, as it better verifies the address codec format. However, this would require referencing complex codec-related code, so how about trying it with reference to this issue: https://github.com/cosmos/cosmos-sdk/issues/20077?

For the test code, it is good as it is, but it would be better if it is structured in a form referenced from [this test code](https://github.com/cosmos/cosmos-sdk/blob/c452721442546da82d199da6d116405819f1b400/x/tx/signing/context_test.go#L17-L146).

Reply:
modified.